cores/esp32/esp32-hal-rgb-led.c

```diff
@@ -0,0 +1,51 @@
+#include "esp32-hal-rgb-led.h"
+
+#ifdef BOARD_HAS_NEOPIXEL
+
+void RGBLedWrite(uint8_t pin, uint8_t red_val, uint8_t green_val, uint8_t blue_val){
+  log_d("RGB: %d %d %d", red_val, green_val, blue_val);
```

Comment:
such logs should be with level "Verbose" instead. Blinking led will print a lot of things :)

Reply:
this was also a forgotten debug output. I will remove it in the next commit.